packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -540,14 +555,53 @@ export class MgtFileList extends MgtTemplatedTaskComponent implements CardSectio
    * @returns {TemplateResult}
    * @memberof mgtFileList
    */
-  protected renderFile(file: DriveItem): TemplateResult {
+  protected renderFile(file: DriveItem | SharedInsight): TemplateResult {
     const view = this.itemView;
-    return (
-      this.renderTemplate('file', { file }, file.id) ||
-      mgtHtml`
-        <mgt-file class="mgt-file-item" .fileDetails=${file} .view=${view}></mgt-file>
-      `
-    );
+    // if file is type SharedInsight, render Shared Insight File
+    if (isSharedInsight(file)) {
+      return this.renderSharedInsightFile(file);
+    } else {
```

Comment:
```suggestion
    }
```
nit: the if has a `return`, no need for else block

Reply:
Applied in the next commit